### PR TITLE
Add edit dialog

### PR DESCRIPTION
**Pull request by sebastiangreger**
Adding an "Edit" option in the Panel list dropdown, opening a dialog modal for editing the commention data.

### CODE REVIEW

File: src/components/Dialogs/EditDialog.vue [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            {value: "repost", text: "Repost"},
+            {value: "bookmark", text: "Bookmark"},
+            {value: "read", text: "Read"},
+            {value: "rsvpyes", text: "RSVP yes"},
+            {value: "rsvpno", text: "RSVP no"},
+            {value: "rsvpmaybe", text: "RSVP maybe"},
+          ],
+        },
+        text: {
+          label: this.$t('commentions.section.edit.text'),
```
Comment (fabianmichael) [inline src/components/Dialogs/EditDialog.vue:93]: This should use a different string, if the HTML Purifier is not available.

File: languages/en.php [hunk 2]
```diff
@@ -68,10 +68,21 @@
...
     // - dialogs
+    'section.edit.author'               => 'Author',
+    'section.edit.name'                 => 'Name',
+    'section.edit.website'              => 'Website URL',
+    'section.edit.email'                => 'Email address',
+    'section.edit.avatar'               => 'Avatar URL',
+    'section.edit.content'              => 'Content',
+    'section.edit.timestamp'            => 'Timestamp',
+    'section.edit.type'                 => 'Type',
+    'section.edit.text'                 => 'Text (HTML/Markdown supported)',
```
Comment (fabianmichael) [inline languages/en.php:84]: … e.g. Text (no formatting supported)

**Comment by sebastiangreger** [comment]
Pragmatic solution: I removed the `(HTML/Markdown supported)` string for now :clown_face: ...too much effort to write a logic to detect the settings, as admins likely know - or quickly learn - how their site works.

#keepitsimple

**Comment by sebastiangreger** [comment]
:heavy_check_mark:  I should really look into automating my build process on commit ;)

**Pull request merged**